src/Time/Formatting.hs

```diff
@@ -0,0 +1,38 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE InstanceSigs        #-}
+{-# LANGUAGE KindSignatures      #-}
+{-# LANGUAGE Rank2Types          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
+{-# LANGUAGE TypeOperators       #-}
+
+-- | This module introduces function to format time in desired way.
+--
+-- Here will be some examples after design is approved.
+
+module Time.Formatting
+       ( Series (..)
+       ) where
+
+import Time.Rational (KnownRat, Rat, withRuntimeDivRat)
+import Time.Units (KnownUnitSymbol, Time, floorUnit, toUnit)
+
+-- | Class for time formatting.
+class Series (units :: [Rat]) where
+    seriesF :: forall unit . (KnownUnitSymbol unit, KnownRat unit)
```

Comment:
I don't actually like duplication of `unit` type variable name I introduced here. And `unit'` is not really good name. Let's replace `unit` and `unit'` in `Time unit` with `someUnit`.